src/core/node/index.ts

```diff
@@ -104,12 +112,31 @@ const LightningRendererNonAnimatingProps = [
   'textAlign',
 ];
 
-export default class Node extends Object {
-  constructor(name) {
+export default class SolidNode extends Object {
+  name: string;
+  lng?: INode;
+  rendered: boolean;
+  autofocus: boolean;
+  zIndex?: number;
+  selected?: number;
+  private _renderProps: any;
+  private _effects: any;
+  private _parent?: SolidNode;
+  private _shader?: ShaderDesc;
+  private _animate?: any;
+  private _style?: any;
+  private _states?: States;
+  private _animationSettings?: any;
+  private _updateLayoutOn?: any;
+  public _isDirty?: boolean; // Public but uses _ prefix
```

Comment:
this probably doesn't need the _ anymore since I switched around how stuff gets forwarded to Lightning